apps/server/src/controllers/user.controller.ts

```
@@ -0,0 +1,126 @@
+import { Request, Response } from "express";
+import { prisma } from "../config/prisma";
+import bcrypt from "bcrypt";
+import jwt from "jsonwebtoken";
+import { config } from "../config/env";
+import type { SignupInput, LoginInput, UpdateProfileInput } from "@split-wisely/validations";
+
+export class UserController {
+    static async signup(req: Request<{}, {}, SignupInput>, res: Response) {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)